http_image_provider/pubspec.yaml

```diff
@@ -14,7 +14,7 @@ environment:
   flutter: ">=3.0.0"
 
 dependencies:
-  http: ^1.1.0
+  http: ^1.2.0
```

Comment:
I don't mind this change, but is there a reason to force consumers to use a more recent version?

Reply:
1.2.0 has a test helper that I'm using. I could just add that constraint to the `dev_dependencies`, if you'd prefer. 1.2.0 has been out for about 5 months.